ramls/examples/patron-block-conditions.sample

```diff
@@ -0,0 +1,59 @@
+{
+  "patronBlockConditions": [
+    {
+      "id": "3d7c52dc-c732-4223-8bf8-e5917801386f",
+      "name": "Maximum number of items charged out",
+      "blockBorrowings": false,
+      "blockRenewals": true,
+      "blockRequests": false,
+      "valueType": "Integer",
+      "message": "The maximum number of charged out items has been reached"
+    },
+    {
+      "id": "72b67965-5b73-4840-bc0b-be8f3f6e047e",
+      "name": "Maximum number of lost items",
+      "blockBorrowings": true,
+      "blockRenewals": true,
+      "blockRequests": true,
+      "valueType": "Integer",
+      "message": "The maximum number of lost items has been reached"
+    },
+    {
+      "id": "584fbd4f-6a34-4730-a6ca-73a6a6a9d845",
+      "name": "Maximum number of overdue items",
+      "blockBorrowings": true,
+      "blockRenewals": true,
+      "blockRequests": true,
+      "valueType": "Integer",
+      "message": "The maximum number of overdue items has been reached"
+    },
+    {
+      "id": "e5b45031-a202-4abb-917b-e1df9346fe2c",
+      "name": "Maximum number of overdue recalls",
+      "blockBorrowings": false,
+      "blockRenewals": false,
+      "blockRequests": false,
+      "valueType": "Integer",
+      "message": "The maximum number of overdue recalls has been reached"
+    },
+    {
+      "id": "cf7a0d5f-a327-4ca1-aa9e-dc55ec006b8a",
+      "name": "Maximum outstanding fee/fine balance",
+      "blockBorrowings": false,
+      "blockRenewals": false,
+      "blockRequests": false,
+      "valueType": "Double",
+      "message": "The maximum outstanding fee/fine balance has been reached"
+    },
+    {
+      "id": "08530ac4-07f2-48e6-9dda-a97bc2bf7053",
+      "name": "Recall overdue by maximum number of days",
+      "blockBorrowings": false,
+      "blockRenewals": false,
+      "blockRequests": false,
+      "valueType": "Integer",
+      "message": "The recall overdue by maximum number of days has been reached"
```

Comment:
Posting this record fails with "One or more blocked actions must be selected for message to be displayed to be used".

Reply:
fixed